### DISCUSSION TITLE
Player Tile Counter

### DISCUSSION THREAD

**Discussion by Romte1** [runelite/runelite#16180] [Ideas]
Hello! i was wondering if it would be possible to add a plugin that displays a counter which shows in ticks for how long you have been standing on the same tile, could be numeric or also with colors similar to visual metronome, but resetting after every move action.

 This plugin would be helpful for players that wanna be precise with their actions during efficient skilling or even pvm as sometimes you need to stand still on the same tile for x ticks. Thanks.

**Comment by Ben-Colwell** [comment]
I'll work on this
